src/interface_r/R/auto_generated_wrappers.R

```diff
@@ -219,199 +215,119 @@ h2o4gpu.gradient_boosting_regressor <- function(
     tree_method = tree_method,
     n_gpus = as.integer(n_gpus),
     predictor = predictor,
-    backend = backend
+    backend = "h2o4gpu"
   )
   h2o4gpu_model(model, c("regressor"))
 }
 
 #' @export
-h2o4gpu.linear_regressor <- function(
-	fit_intercept = TRUE,
-	normalize = FALSE,
-	copy_X = TRUE,
-	n_jobs = 1L,
+h2o4gpu.elastic_net_regressor <- function(
+	n_threads = NULL,
+	gpu_id = 0L,
 	n_gpus = -1L,
-	tol = 0.0001,
+	fit_intercept = TRUE,
+	lambda_min_ratio = 1e-07,
+	n_lambdas = 100L,
+	n_folds = 5L,
+	n_alphas = 5L,
+	tol = 0.01,
+	tol_seek_factor = 0.1,
+	lambda_stop_early = TRUE,
 	glm_stop_early = TRUE,
 	glm_stop_early_error_fraction = 1.0,
-	verbose = FALSE,
-	backend = "auto") {
-
-  model <- h2o4gpu$LinearRegression(
-    fit_intercept = fit_intercept,
-    normalize = normalize,
-    copy_X = copy_X,
-    n_jobs = as.integer(n_jobs),
-    n_gpus = as.integer(n_gpus),
-    tol = tol,
-    glm_stop_early = glm_stop_early,
-    glm_stop_early_error_fraction = glm_stop_early_error_fraction,
-    verbose = verbose,
-    backend = backend
-  )
-  h2o4gpu_model(model, c("regressor"))
-}
-
-#' @export
-h2o4gpu.logistic_regressor <- function(
-	penalty = "l2",
-	dual = FALSE,
-	tol = 0.01,
-	C = 1.0,
-	fit_intercept = TRUE,
-	intercept_scaling = 1.0,
-	class_weight = NULL,
-	random_state = NULL,
-	solver = "liblinear",
 	max_iter = 5000L,
-	multi_class = "ovr",
 	verbose = 0L,
-	warm_start = FALSE,
-	n_jobs = 1L,
-	n_gpus = -1L,
-	glm_stop_early = TRUE,
-	glm_stop_early_error_fraction = 1.0,
-	backend = "auto") {
+	 store_full_path = 0L,
+	lambda_max = NULL,
+	alpha_max = 1.0,
+	alpha_min = 0.0,
+	alphas = NULL,
+	lambdas = NULL,
+	double_precision = NULL,
+	order = NULL) {
 
-  model <- h2o4gpu$LogisticRegression(
-    penalty = penalty,
-    dual = dual,
-    tol = tol,
-    C = C,
-    fit_intercept = fit_intercept,
-    intercept_scaling = intercept_scaling,
-    class_weight = class_weight,
-    random_state = as_nullable_integer(random_state),
-    solver = solver,
-    max_iter = as.integer(max_iter),
-    multi_class = multi_class,
-    verbose = as.integer(verbose),
-    warm_start = warm_start,
-    n_jobs = as.integer(n_jobs),
+  model <- h2o4gpu$ElasticNetH2O(
+    n_threads = n_threads,
```

Comment:
Should be `as_nullable_integer(n_threads)`.  See other comment about this on how to fix.